sdks/java/core/src/main/proto/README.md

```diff
@@ -1,7 +1,25 @@
-## Protocol Buffers in Google Cloud Dataflow
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
 
-This directory contains the Protocol Buffer messages used in Google Cloud
-Dataflow.
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+## Protocol Buffers in Apache Beam
```

Comment:
It looks like the only thing here ([proto2_coder_test_messages.proto](https://github.com/jbonofre/incubator-beam/blob/BEAM-254-RAT/sdks/java/core/src/main/proto/proto2_coder_test_messages.proto)) is for testing. If that's the case, I would suggest updating the verbage of this README with the test details and removing dataflow-specific bits. At any rate, it might be easier to save for a separate PR as to not hold up this one.

 Adding a few people that might know more: @dhalperi @kennknowles